app/views/posts/post_list.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Posts#list</h1>
+<p>Find me in app/views/posts/post_list.html.erb</p>
```

Comment:
- [**Optional**] It is preferable to always name your views files in correspondence with their actions. Naming this file as `index.html.erb` shows that this is the view file for the `index` action in the controller. I think it also avoids confusion in case you need to make any changes in the future. If you prefer to go in this manner, please rename all your view files. 

Thanks :pray: